tools/lsp/wasm_main.rs

```diff
@@ -295,6 +295,29 @@ impl SlintServer {
                 let edit = crate::language::update_element(&self.ctx, position, properties);
                 let sn = self.ctx.server_notifier.clone();
 
+                if let Ok(edit) = edit {
+                    wasm_bindgen_futures::spawn_local(async move {
+                        let fut = sn.send_request::<lsp_types::request::ApplyWorkspaceEdit>(
```

Comment:
This is duplicated 3 times already.  Maybe worth a apply_workspace_edit function.

In fact, this is also duplicated with main.rs.  Why can't language::update_element do that?

Reply:
You can, for example have a function defined both in main.rs and wasm_main.rs, so you can call `crate::spawn_future`